Mathlib/Algebra/Ring/BooleanRing.lean

```diff
@@ -56,14 +56,12 @@ variable [BooleanRing α] (a b : α)
 instance : IsIdempotent α (· * ·) :=
   ⟨BooleanRing.mul_self⟩
 
--- Porting note: simpNF linter complains. This causes lemmas in other files not to be in simpNF
```

Comment:
how could this possibly fix `simpNF`?

Reply:
The change to Mathlib/Algebra/PUnitInstances.lean is what makes the linter not complain. This is helpful to make it clear that all the `norm_num [blah]` calls that worked in mathlib 3 still work here